circle.yml

```diff
@@ -67,7 +67,10 @@ deployment:
       - |
         test -z "${DOCKER_USER}" || (
           docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS &&
-          (test "${DOCKER_ORGANIZATION:-$DOCKER_USER}" == "weaveworks" || docker tag weaveworks/scope:latest ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:latest) &&
+          (test "${DOCKER_ORGANIZATION:-$DOCKER_USER}" == "weaveworks" || (
+            docker tag weaveworks/scope:latest ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:latest &&
+            docker tag weaveworks/scope:$(./image-tag) ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:$(./image-tag)
+          )) &&
           docker push ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope &&
           docker push ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:$(./image-tag) &&
           wcloud deploy ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:$(./image-tag)
```

Comment:
Okay I think it might be worth adding a owner section for weaveworks (see https://circleci.com/docs/configuration/#deployment) and another for everyone else, which does the org move.